### ISSUE TITLE
Kilosort version and Wiki doesn't show images

### ISSUE THREAD

**Issue by lkatsouri** [MouseLand/Kilosort#252]
Dear Marius, 

Thank you so much for releasing one more update on Kilosrt2. I have been using it for tetrodes’ recordings  and I downloaded Kilosort2 sometime last December.
I am probably a bit slow but the release says v2.0 and the text below says ‘Welcome to v2.5’. Which version is it on Github at the moment? And what’s the best way to download it, replace everything in my Kilosort with the zip file or update it using github?

On another note, it appears that the images on the Wiki are broken, they don’t appear on my Mac PC neither on Safari or Chrome.
 
Thank you so much for everything you are doing and I am looking forward to your response.
 
Best wishes,
 
Loukia
<img width="875" alt="Screenshot 2020-10-31 at 11 27 16" src="https://user-images.githubusercontent.com/36635672/97778017-105ed380-1b6c-11eb-858e-8294a962c8b9.png">

**Comment by marius10p** [comment]
Thanks Loukia, I fixed the links, although I will need to do a more thorough update of some parts of the wiki, as they only apply to Kilosort 2.0. 

The current version on Github is, and always will be, the latest version. What you are seeing on the github are the past "releases", meaning I made an archive of the 2.0 version for people who need to get it back and use that instead of 2.5. The best way to update is either through git (git pull), or just delete the old version and download a zip file again. Remember to run mexGPUall again to compile the new CUDA files.
